wrappers/winuser.go

```diff
@@ -326,6 +362,30 @@ func GetWindowThreadProcessId(hwnd syscall.Handle, processID *uint32) (uint32, e
 	return uint32(r1), nil
 }
 
+func EnumDesktops(winsta syscall.Handle, fnEnum DESKTOPENUMPROC, param uintptr) error {
+	fnEnumRaw := func(desktop *uint16, lparam uintptr) uintptr {
+		return uintptr(fnEnum(
+			desktop,
+			lparam))
```

Comment:
How about if we have `DESKTOPENUMPROC` return a `bool` and handle the conversion here?

Reply:
I am not sure because in this case we will lose information about what error has occurred.  Sometimes it can be useful ...

Reply:
So, the Win32 API documentation has the callback returning a `BOOL`, so anything other than `TRUE` or `FALSE` would be undocumented behavior.  See https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/ms682612(v%3Dvs.85).

Reply:
Oh ... I was thinking about the EnumDesktops procedure, not about callback. So of course you right. Changed. 
Also I changed EnumDesktop return value in case of error. Sometimes it may be needed to cancel enumeration but keep current results. How do you think about it ?